exporters/otlp/common/src/jmh/java/io/opentelemetry/exporter/internal/otlp/RequestMarshalBenchmarks.java

```diff
@@ -24,29 +25,72 @@
 @Fork(1)
 public class RequestMarshalBenchmarks {
 
+  private static final LowAllocationTraceRequestMarshaler MARSHALER =
+      new LowAllocationTraceRequestMarshaler();
+  private static final TestOutputStream OUTPUT = new TestOutputStream();
+
   @Benchmark
   @Threads(1)
-  public TestOutputStream createCustomMarshal(RequestMarshalState state) {
+  public int createCustomMarshal(RequestMarshalState state) {
     TraceRequestMarshaler requestMarshaler = TraceRequestMarshaler.create(state.spanDataList);
-    return new TestOutputStream(requestMarshaler.getBinarySerializedSize());
+    return requestMarshaler.getBinarySerializedSize();
   }
 
   @Benchmark
   @Threads(1)
-  public TestOutputStream marshalCustom(RequestMarshalState state) throws IOException {
+  public int marshalCustom(RequestMarshalState state) throws IOException {
```

Comment:
Any idea what the name "custom" means here? Could be a reference to the "hand rolled" marshalling which we probably compared to the protobuf library based marshaling. Consider renaming to reflect the current state of things. We have marshalers which are stateless and stateful, and we serialize to protobuf binary and to protobuf json.

Reply:
> Could be a reference to the "hand rolled" marshalling which we probably compared to the protobuf library based marshaling.

You are right. These benchmarks were introduced in https://github.com/open-telemetry/opentelemetry-java/pull/2304 and compare custom vs proto. By now the original proto code has been removed.